code/src/vs/server/node/che/remoteTerminalMachineExecChannel.ts

```diff
@@ -158,7 +158,7 @@ export class RemoteTerminalMachineExecChannel implements IServerChannel<RemoteAg
 		}
 
 		this.logService.error(`RemoteTerminalChannel: unsupported command/${command}`);
-		return undefined;
+		return {};
```

Comment:
I think we should return {} only if command name is $getEnvironment to address the real issue that was for getEnvironment but why not return an empty object instead undefined for unsupported case anyway